example/wangsets.go

```diff
@@ -0,0 +1,29 @@
+package main
```

Comment:
this sample should be included as godoc comment for `GetWangColors` function

Reply:
Same as above